indico_toolkit/indico_wrapper/metrics.py

```diff
@@ -0,0 +1,104 @@
+import pandas as pd
+from typing import List
+from indico import IndicoClient
+
+from .indico_wrapper import IndicoWrapper
+from indico_toolkit import ToolkitInputError
+
+
+class ExtractionMetrics(IndicoWrapper):
```

Comment:
two thoughts:
-do we need a generic metrics class that can get metrics from any species of model?
-could we infer the model type from what is returned by the get_metrics call and have that be a property of the Metrics class?

this is probably not important now, but could be once workflows become more generic.

Reply:
I already had the extraction formatting code written (that's why starting with that model type). Call takes a "model group ID" so user should know what type the model is ahead of time (and theoretically could use ExtractionMetrics or ClassificationMetrics, but obviously you could build it into a generic Metrics class where a property is set for model type based on query response format/content)